docs/src/modules/components/AppLayoutDocs.js

```diff
@@ -138,13 +144,7 @@ export default function AppLayoutDocs(props) {
 
   return (
     <Layout {...layoutProps}>
-      <GlobalStyles
-        styles={{
-          ':root': {
```

Comment:
The variables are not being added when using the MUI's GlobalStyles, but they are added when using the `createGlobalStyle` util directly. We'll need to investigate this separately - it's not related to v6 tough.